core/sqf/monitor/linux/zclient.cxx

```diff
@@ -53,6 +53,9 @@
 // The monitors register their znodes under the cluster znode
 #define ZCLIENT_CLUSTER_ZNODE               "/cluster"
 
+// zookeeper connection retry count
+#define ZOOKEEPER_RETRY_COUNT                3
```

Comment:
Based on the test result, actually retry for once is enough for this case. So it's not quite necessary to modify this value.